codespell_lib/data/dictionary.txt

```diff
@@ -269,6 +269,8 @@ addtional->additional
 addtionally->additionally
 adecuate->adequate
 adequit->adequate
+adge->edge, badge,
+adges->edges, badges,
```

Comment:
sorry realized one more, `adage` and `adages`

Reply:
(probably at the end of these option lists)

Reply:
Good catch! added and squashed commits